src/com/goide/inspections/GoStructInitializationInspection.java

```diff
@@ -35,60 +35,50 @@
 
 public class GoStructInitializationInspection extends GoInspectionBase {
   public static final String REPLACE_WITH_NAMED_STRUCT_FIELD_FIX_NAME = "Replace with named struct field";
-  public boolean reportImportedStructs;
+  public boolean reportLocalStructs;
```

Comment:
I can do the inspection without changing this if it's a breaking change for the users.